### PR TITLE
Enable qualcomm LAVA lab

### PR DESCRIPTION
**Pull request by JenySadadia**
Closes: https://github.com/kernelci/kernelci-project/issues/358

- config/pipeline.yaml: add lab config for Qualcomm
- config/pipeline.yaml: add `baseline-x86` job for qualcomm
- docker-compose.yaml: add lab-qualcomm runtime
- config/pipeline.yaml: add `baseline-arm64` job for qualcomm

### CODE REVIEW

**Review by pawiecz — APPROVED** [review]

File: config/pipeline.yaml [hunk 1]
```diff
@@ -646,6 +667,17 @@ scheduler:
...
+  - job: baseline-x86-qualcomm
+    event:
+      channel: node
+      name: kbuild-gcc-10-x86
+      result: pass
+    runtime:
+      type: lava
+      name: lava-qualcomm
+    platforms:
+      - qemu
```
[the remark this answers is not in the document]
Comment (mwasilew) [reply config/pipeline.yaml:679]: I disabled qemu-aarch64-01 device some time ago, so it shouldn't be an issue.

**Comment by mwasilew** [comment]
@JenySadadia many thanks for that. It looks like it's working (there are a couple of jobs running). I guess next step for me is to prepare job templates for the devices I really would like to test and update the filters in the pipeline.

**Comment by JenySadadia** [comment]
Hi @mwasilew 
I have enabled baseline tests for `qemu` and `bcm2711-rpi-4-b` platforms atm.
There seems like some infra issue for the arm64 job i.e. https://lava.infra.foundries.io/scheduler/job/43051.

> I guess next step for me is to prepare job templates for the devices I really would like to test and update the filters in the pipeline.

Please let us know if we can help you with that.

**Comment by mwasilew** [comment]
> Hi @mwasilew I have enabled baseline tests for `qemu` and `bcm2711-rpi-4-b` platforms atm. There seems like some infra issue for the arm64 job i.e. https://lava.infra.foundries.io/scheduler/job/43051.

It requires TFTP and I don't think I ever set it up. It's best to disable RPi4 kernelci jobs in this case.

> 
> > I guess next step for me is to prepare job templates for the devices I really would like to test and update the filters in the pipeline.
> 
> Please let us know if we can help you with that.

I have example job that boots kernelci build:
https://lava.infra.foundries.io/scheduler/job/43040/definition

Looking at the available templates I wasn't able to find a similar one. I'm now trying to setup my local kernelci env so I can send a PR to fix it.

qcs6490-rb3g2 is the only device that should be used for kernelci at the moment.

**Comment by JenySadadia** [comment]
> It requires TFTP and I don't think I ever set it up. It's best to disable RPi4 kernelci jobs in this case.

Okay. I disabled the job.

**Comment by mwasilew** [comment]
> Okay. I disabled the job.

You were quicker. I was planning on enabling TFTP today :) I'll put it on backburner.

**Comment by JenySadadia** [comment]
> > Okay. I disabled the job.
> 
> You were quicker. I was planning on enabling TFTP today :) I'll put it on backburner.

Oh, no worries. I just need to uncomment a line once you enable TFTP.

**Comment by JenySadadia** [comment]
> I enabled tftp now: https://lava.infra.foundries.io/scheduler/job/43135 With dockerized setup it was harder than I expected.

That was quick. Thanks, I'll enable the test.

> @JenySadadia since I'm receiving some test jobs from kernelci I assume this config is already deployed somewhere. Is this some kind of staging environment?

Yes, we have services deployed on a staging instance. Here is the link if you want to check https://staging.kernelci.org:9088/.

**Pull request merged**